command/tls/cert/create/tls_cert_create.go

```diff
@@ -86,16 +89,22 @@ func (c *cmd) Run(args []string) int {
 		}
 	}
 
+	for _, i := range c.ipaddresses {
+		if len(i) > 0 {
+			IPAddresses = append(IPAddresses, net.ParseIP(strings.TrimSpace(i)))
+		}
+	}
+
 	if c.server {
 		name = fmt.Sprintf("server.%s.%s", c.dc, c.domain)
 		DNSNames = append(DNSNames, []string{name, "localhost"}...)
-		IPAddresses = []net.IP{net.ParseIP("127.0.0.1")}
+		IPAddresses = append(IPAddresses, []net.IP{net.ParseIP("127.0.0.1")}...)
```

Comment:
```suggestion
		IPAddresses = append(IPAddresses, net.ParseIP("127.0.0.1"))
```

Reply:
I should have been more attentive when reading the go doc :/ 
Thank you!